blueprints/module-unification-app/files/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Should be sync'ed with app's `.eslintrc.js` file (we use `overrides` and such now)...